### DISCUSSION TITLE
Website not showing theme

### DISCUSSION THREAD

**Discussion by SoffiMeower** [community/community#22313] [Other Feature Feedback, Questions, & Ideas]
<p>I’ve done all of the steps shown here: <a href="https://github.com/mmistakes/minimal-mistakes#gem-based-method" rel="nofollow noopener">https://github.com/mmistakes/minimal-mistakes#gem-based-method</a> and copied the new Gemfile(s) over to github. Please help. <a href="https://github.com/revelation-studios/Revelation-Website" rel="nofollow noopener">Github files</a></p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hey <a class="mention" href="/ghostslayer989">@ghostslayer989</a>,</p>
<p>Thanks for being here! Have you tried working/directly copying all of the theme files into your project? I don’t see them in your repo. Is possible someone here has used this theme before and can help but if you dont’t receive any feedback you could try opening an issue on the projects repo directly.</p>

**Comment by SoffiMeower** [comment]
<p>I just scratched jekyll out and am doing html with Bulma CSS-(Also on Github!).</p>

**Comment by AndreaGriffiths11** [comment]
<p>That works too! I’m glad you got it going.</p>
